Polyhedron/demo/Polyhedron/CMakeLists.txt

```diff
@@ -268,6 +268,14 @@ if(CGAL_Qt5_FOUND AND Qt5_FOUND)
   endif()
   add_item(scene_tetrahedra_item Scene_tetrahedra_item.cpp)
   target_link_libraries(scene_tetrahedra_item PUBLIC scene_c3t3_item)
+
+  find_package(ITK QUIET NO_MODULE)
+  if (ITK_FOUND)
+    include(${ITK_USE_FILE})
+    target_link_libraries(scene_c3t3_item PUBLIC ${ITK_LIBRARIES})
+    target_compile_definitions(scene_c3t3_item PUBLIC CGAL_USE_ITK)
```

Comment:
`scene_c3t3_item` does not use ITK. That is `mesh_3_plugin` that uses it.

Reply:
I'm already working on that :-)